pymepixviewer/panels/blobview.py

```diff
@@ -299,6 +299,8 @@ def onCentroid(self, event):
             self.updateBlobData(cluster_shot, cluster_x, cluster_y, cluster_tof)
 
     def onEvent(self, event):
+        print('IN ONEVENT')
```

Comment:
This should probably be `logging` if the output is needed.

Reply:
Outdated.

Reply:
Sorry, I didn't realize, I was looking at code from an old commit.